apps/web/src/pages/blog/[slug].module.css

```
@@ -92,6 +92,27 @@
   border-radius: 0.75rem;
 }
 
+.content :global(.mermaid) {
```

Stylelint (17.3.0)

[error] Unexpected unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)